base/frontend/sourcegraph-frontend.Deployment.yaml

```diff
@@ -27,49 +27,27 @@ spec:
         app: sourcegraph-frontend
         deploy: sourcegraph
     spec:
+      initContainers:
+      - name: migrator
+        image: sourcegraph/migrator:3.35.1
+        args: ['up']
+        envFrom:
+        - configMapRef:
+            name: sourcegraph-frontend-environment
       containers:
       - name: frontend
         image: index.docker.io/sourcegraph/frontend:insiders@sha256:f6b962027b6cdb4a1cdea2573d9dfd89f353e2efb5ac2c485bf086bfa01e7a93
         args:
         - serve
         env:
-        - name: PGDATABASE
-          value: sg
-        - name: PGHOST
-          value: pgsql
-        - name: PGPORT
-          value: "5432"
-        - name: PGSSLMODE
-          value: disable
-        - name: PGUSER
-          value: sg
-        - name: CODEINSIGHTS_PGDATASOURCE
-          value: postgres://postgres:password@codeinsights-db:5432/postgres
-        - name: CODEINTEL_PGDATABASE
-          value: sg
-        - name: CODEINTEL_PGHOST
-          value: codeintel-db
-        - name: CODEINTEL_PGPORT
-          value: "5432"
-        - name: CODEINTEL_PGSSLMODE
-          value: disable
-        - name: CODEINTEL_PGUSER
-          value: sg
         # POD_NAME is used by CACHE_DIR
         - name: POD_NAME
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
-        # CACHE_DIR stores larger items we cache. Majority of it is zip
-        # archives of repositories at a commit.
-        - name: CACHE_DIR
-          value: /mnt/cache/$(POD_NAME)
-        - name: GRAFANA_SERVER_URL
-          value: http://grafana:30070
-        - name: JAEGER_SERVER_URL
-          value: http://jaeger-query:16686
-        - name: PROMETHEUS_URL
-          value: http://prometheus:30090
+        envFrom:
```

Comment:
This change ca be risky, as for now, every change in ENVs causes new rollout of frontends for clients. I think that with `envFrom` this behaviour might change, as only configMap will be changed, and it does not cause rollout.